homeworks/react-continued-3/rosyurch/src/Info.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react'
+
+function Info (props) {
+    const [isOpen, setIsOpen] = useState(false);
+
+    function toggle () {
+        setIsOpen(!isOpen);
```

Comment:
if your next state value based on current you should use function `currentState => !currentState`